controllers/city.controller.js

```diff
@@ -0,0 +1,39 @@
+const City = require("../models/city.model");
```

Comment:
You should refactor this to use the express middleware model.  It will probably be easier to manage in the future.
https://expressjs.com/en/guide/using-middleware.html